src/components/01-branding/02-typography/heading/heading-default.config.yml

```diff
@@ -0,0 +1,18 @@
+label: Heading
```

Comment:
heading-default.config.yml filename should just be heading.config.yml. After the change having the label set for the base or any of the variants will be unnecessary